pkg/git/scm_git.go

```diff
@@ -0,0 +1,58 @@
+package git
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/jenkins-x/go-scm/scm"
+	"github.com/jenkins-x/go-scm/scm/factory"
+)
+
+// SCMClient defines the ability to create a pull request on a remote repository.
```

Comment:
what does SCM stand for? Its always acronym'd in the code :smiley:

Reply:
Source Control Management :D